plugin/src/main/java/io/jenkins/plugins/casc/core/GlobalNodePropertiesConfigurator.java

```diff
@@ -0,0 +1,61 @@
+package io.jenkins.plugins.casc.core;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+import hudson.slaves.EnvironmentVariablesNodeProperty.Entry;
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.ConfiguratorException;
+import io.jenkins.plugins.casc.impl.configurators.DataBoundConfigurator;
+import io.jenkins.plugins.casc.model.CNode;
+import io.jenkins.plugins.casc.model.Mapping;
+import java.util.ArrayList;
+import java.util.List;
+
+@Extension
+public class GlobalNodePropertiesConfigurator extends DataBoundConfigurator<EnvironmentVariablesNodeProperty> {
```

Comment:
IMO this is the wrong fix and it is https://github.com/jenkinsci/configuration-as-code-plugin/blob/148f87d74b1e1623babf4b4b3376c879acff35a4/plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java#L660-L662 which should be corrected. If a field is set to the empty string (rather than null), we should export

```yaml
thatField: ''
```

regardless of where it is. If some config class neglects to call `Util.fixEmpty` (or `fixEmptyAndTrim` etc.) when processing a parameter to a `@DataBoundSetter` which cannot logically be the empty string, then fix that setter.

Reply:
> which might mean to fix how many plugins?

Not sure, but they need to be fixed anyway even without JCasC.